pytest.ini

```diff
@@ -70,3 +70,5 @@ filterwarnings =
     ignore:pkg_resources is deprecated as an API.*:DeprecationWarning
     ignore:Deprecated call to .*pkg_resources\.declare_namespace.*mpl_toolkits.*:DeprecationWarning
     ignore:Extension .*sunpy-1\.0\.0:asdf.exceptions.AsdfManifestURIMismatchWarning
+    # This warning should go away in asdf-astropy 0.7.1 and we can stop being ignored then
+    ignore::asdf.exceptions.AsdfPackageVersionWarning
```

Comment:
FYI: 0.7.1 with the fix is up on pypi. https://pypi.org/project/asdf-astropy/0.7.1/